### DISCUSSION TITLE
Cron

### DISCUSSION THREAD

**Discussion by pauloricardoferreira** [apache/airflow#38775] [Q&A]
### Description

I would like to request something for Scheduler via Cron.

in that case:

*/27 8-11,14-18 * * 1-5

it should run at these times

8 o'clock
08:27
08:54
09:21
09:48
10:15
10:42

What is happening is that when you run the task at 08:54, it runs in the next full hour.

Could it be a bug?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by Taragolis** [comment]
> it should run at these times

> Could it be a bug?

It should run after `date_interval_end` if you use default [CronDataIntervalTimetable](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/timetable.html#crondataintervaltimetable)


- [DAG Runs: Data Interval ](https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/dag-run.html#data-interval)
- [Scheduler](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/scheduler.html)
- [FAQ: What does `execution_date` mean?](https://airflow.apache.org/docs/apache-airflow/stable/faq.html#faq-what-does-execution-date-mean)
